lib/board.rb

```diff
@@ -1,5 +1,7 @@
 class Board
   attr_reader :board
+  attr_reader :POSITION_MIN
+  attr_reader :POSITION_MAX
```

Comment:
Its because attr_reader means 'read only' and so you would not be able to re-assign the counter with a new value. There is attr_writer as well. There is a short description here that might help: https://www.codecademy.com/en/forum_questions/50f0192b102455349200372d